recipes/xmscore/meta.yaml

```diff
@@ -19,12 +19,12 @@ requirements:
     - cmake >=3.0.0
     - boost
     - pybind11
-    - python =3.6.*
+    - python
   host:
-    - python =3.6.*
+    - python
     - pip
   run:
-    - python =3.6.*
```

Comment:
Looks like the code upstream is limited to 3.6. See https://github.com/Aquaveo/xmscore/blob/ccf048f99de393af06e2571294557e86511ca79a/_package/setup.py#L26

Reply:
yeah, i was just looking at that. It might not have to be there upstream, but it is right now, and i won't be able to update that requirement for a couple of weeks